05_stringer/madhavansu/main_test.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+
+	// When
+	main()
+
+	// Then
+	expected := strconv.Quote(`127.0.0.1`)
```

Comment:
Same as above `strconv.Quote` is not required

Reply:
This is for the readability purpose, as a user we will know what is expected and what is the actual value.